Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM lambci/lambda:nodejs10.x
+COPY node_modules/ /var/task/node_modules
```

Comment:
Change this to run the install step eg:
```suggestion
WORKDIR /var/task
RUN npm install
```

Reply:
I'm wrong again - however we can use a pre_build hook to do the same thing as `npm run build-node-modules`